Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9.0-alpine3.12
```

Comment:
This should use 3.8 at most, preferably 3.7

Reply:
Misread requirements in README, will change